packages/@guardian/source-react-components/src/icons/SvgSpinner.tsx

```diff
@@ -0,0 +1,61 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { brand, iconSize } from '@guardian/source-foundations';
+import type { IconProps } from './types';
+
+interface LoadingCircleIconProps extends IconProps {
+	additionalCss?: SerializedStyles;
+}
+
+const circleLineThickness = 50;
+
+const lightblueStyles = css`
+	stroke: #a5c0e8;
+	stroke-width: ${circleLineThickness};
+	fill: transparent;
+`;
+const darkblueStyles = css`
+	stroke: ${brand['400']};
+	stroke-dasharray: 820;
+	stroke-dashoffset: 820;
+	stroke-width: ${circleLineThickness};
+	fill: transparent;
+`;
+
+export const SvgSpinner = ({
+	size,
+	additionalCss,
+}: LoadingCircleIconProps): EmotionJSX.Element => {
+	return (
+		<svg
+			width={size ? iconSize[size] : undefined}
+			viewBox="0 0 300 300"
```

Comment:
Is there an easy way to convert this to 30x30 for consistency with other icons?

Reply:
had a go, and it seems to work alright!